trader/trader.go

```diff
@@ -615,30 +616,29 @@ func (t *Trader) setExistingOffers(sellingAOffers []hProtocol.Offer, buyingAOffe
 	t.sellingAOffers, t.buyingAOffers = sellingAOffers, buyingAOffers
 }
 
-func (t *Trader) countOfferChangeTypes(offers []txnbuild.Operation) (numCreate int, numUpdate int, numDelete int, e error) {
-	var offerXdr xdr.Operation
+func countOfferChangeTypes(offers []*txnbuild.ManageSellOffer) (int, int, int, error) {
+	numCreate, numDelete, numUpdate := 0, 0, 0
```

Comment:
it's ok to leave the signature as `(numCreate int, numUpdate int, numDelete int, e error)` so we know what each returned value is. makes it easy for readers/consumers of the function and when figuring out what to return.

just because we are explicitly returning the values doesn't mean we cannot have named variables in the return types. 
alternatively we an use this approach for the return types `(/*numCreate*/ int, /*numUpdate*/ int, /*numDelete*/ int, error)` (error doesn't need a comment in this case)
we use both of the above patterns in a few parts of the codebase

Reply:
note that in the previous commit this was create, update, delete, and now we return create, delete, update -- this is very easy to do because the types are same which is where the above convention helps just a little

Reply:
Great suggestion! I changed it to create-delete-update to be in alphabetical order - that felt the least arbitrary - but definitely implementing this approach to naming.